src/config/detekt/baseline-main.yml

```diff
@@ -2319,6 +2319,7 @@
     <ID>MaxLineLength:UnableToConnectToDatabaseError.kt$UnableToConnectToDatabaseError$org.vaccineimpact.orderlyweb.models.ErrorInfo("database-connection-error", "Unable to establish connection to the database at $url")</ID>
     <ID>MaxLineLength:UnknownObjectError.kt$UnknownObjectError$org.vaccineimpact.orderlyweb.models.ErrorInfo("unknown-${mangleTypeName(typeName)}", "Unknown ${mangleTypeName(typeName)} : '$id'")</ID>
     <ID>MaxLineLength:VersionRouteConfig.kt$VersionRouteConfig$APIEndpoint("/reports/:name/versions/:version/changelog/", versionController, "getChangelogByNameAndVersion")</ID>
+    <ID>MaxLineLength:org:vaccineimpact:orderlyweb:viewmodels:IndexViewModel:kt:100</ID>
```

Comment:
This and several of the other new entries in this file appear to relate to new code - which shouldn't need to be excluded from style checks